src/identity/events/multiOrgEvents.ts

```diff
@@ -12,6 +12,30 @@ export enum HeaderNavEvent {
   UserAvatarClick = 'headerNav.userAvatarIcon.clicked',
   UserProfileClick = 'headerNav.userAvatarProfile.clicked',
   UserLogoutClick = 'headerNav.userAvatarLogOut.clicked',
+
+  CreateOrgClick = 'headerNav.createOrg.clicked',
+  UpgradeButtonClick = 'headerNav.upgradeButton.clicked',
+}
+
+export enum CreateOrgOverlayEvent {
+  OrgCreationSuccess = 'headerNav.createOrgOverlay.creationSuccess',
+  OrgCreationFail = 'headerNav.createOrgOverlay.creationFail',
+  SwitchToNewOrg = 'headerNav.createOrg.switchedToNewOrg',
```

Comment:
I think this one needs to be `headerNav.createOrgOverlay.switchedToNewOrg`

Reply:
I might also consider alphabetizing the enums in this file, now that it's growing larger.